### PR TITLE
[ASM] EXPANDR-3608: store potential offending firewall rules

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-3608

## Description
Updated the playbook to incorporate the outputs of *GCPOffendingFirewallRule* script to store potential offending GCP firewall rules.

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , please review when possible

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment_README.md [hunk 1]
```diff
@@ -14,13 +14,12 @@ This playbook does not use any integrations.
 
 ### Scripts
 
-* GridFieldSetup
 * Set
-* GetTime
+* GridFieldSetup
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment_README.md:18]:
```suggestion
GridFieldSetup
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment_README.md [hunk 2]
```diff
@@ -14,13 +14,12 @@ This playbook does not use any integrations.
...
 
 ### Commands
 
-This playbook does not use any commands.
+* setAlert
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment_README.md:22]:
```suggestion
setAlert
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_28.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex ASM - GCP Enrichment
+
+- Updated the playbook to incorporate the outputs of *GCPOffendingFirewallRule* script to store potential offending GCP firewall rules.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_28.md:6]:
```suggestion
Updated the playbook to incorporate the outputs of the **GCPOffendingFirewallRule** script to store potential offending GCP firewall rules.
```

**Comment by ShirleyDenkberg** [comment]
@melamedbn @yucohen Doc review completed.

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml [hunk 4]
```diff
@@ -709,54 +717,7 @@ tasks:
...
-    isoversize: false
-    isautoswitchedtoquietmode: false
-  "21":
-    id: "21"
-    taskid: d747cea6-9883-4960-8c1a-5bb9f22fbeb7
-    type: regular
-    task:
-      id: d747cea6-9883-4960-8c1a-5bb9f22fbeb7
-      version: -1
-      name: Set system IDs grid field (firewall)
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml:729]: Is there a reason not to store this anymore? Would it still not be useful information for the customer?

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml [hunk 5]
```diff
@@ -2297,6 +2258,94 @@ tasks:
...
+      - "33"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      gridfield:
+        simple: asmsystemids
+      keys:
+        simple: type,id,link
+      val1:
+        simple: GCP-Offending-FW-Rule
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml:2321]: Can this be a more generic name, if there are not multiple rules for multiple cloud providers in an alert?

**Review by BigEasyJ — COMMENTED** [review]
Will approve after discussions, other than the outstanding questions the logic looks good!

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml (hunk 4 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml:729]: So in https://github.com/demisto/content/pull/32678 i am actually replacing the !gcp-compute-list-firewall command with the GCPOffendingFirewallRule automation that uses that command.  Before we were storing all GCP firewall rules in a particular network in that system IDs gridfield which I don't think is useful.

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml (hunk 5 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml:2321]: any recommendations? I am open to suggestions.

Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml:2321]: `Offending-Network-Rule`?

Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml:2321]: For NGFW we do FIREWALL-RULE-NAME, maybe keep the same as that?

Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml:2321]: `FIREWALL-RULE-NAME` works if we only use is for offending rules.

**Review by BigEasyJ — APPROVED** [review]

**Comment by johnnywilkes** [comment]
still waiting on https://github.com/demisto/content/pull/32795

**Review by yucohen — APPROVED** [review]

**Pull request merged**